src/view/components/overrides.jsx

```diff
@@ -0,0 +1,273 @@
+ * @returns {React.Element}
+ */
+const Overrides = ({
+  instanceFieldName,
+  largeHeader = false,
+  showFields = [...Object.values(FIELD_NAMES)]
+}) => {
+  const prefix = instanceFieldName
+    ? `${instanceFieldName}.edgeConfigOverrides`
+    : "edgeConfigOverrides";
+  const showFieldsSet = new Set(showFields);
+
+  return (
+    <>
+      <HeaderContainer largeHeader={largeHeader}>
+        Datastream Configuration Overrides
+      </HeaderContainer>
+      <FormElementContainer>
+        <FieldSubset>
+          <Flex direction="column">
+            {showFieldsSet.has(FIELD_NAMES.eventDatasetOverride) && (
+              <DataElementSelector>
+                <FormikTextField
+                  data-test-id={FIELD_NAMES.eventDatasetOverride}
+                  label="Event dataset"
+                  name={`${prefix}.com_adobe_experience_platform.datasets.event.datasetId`}
+                  description="The ID for the destination event dataset in the Adobe Experience Platform. The dataset set here overrides the one set in your datastream configuration."
+                  width="size-5000"
+                />
+              </DataElementSelector>
+            )}
+            {showFieldsSet.has(FIELD_NAMES.idSyncContainerOverride) && (
+              <DataElementSelector>
+                <FormikTextField
+                  data-test-id={FIELD_NAMES.idSyncContainerOverride}
+                  label="Third-party ID sync container"
+                  name={`${prefix}.com_adobe_identity.idSyncContainerId`}
+                  inputMode="numeric"
+                  pattern={/\d+/}
+                  description="The ID for the destination third-party ID sync container in Adobe Audience Manager. The container set here overrides the one set in your datastream configuration."
+                  width="size-5000"
+                />
+              </DataElementSelector>
+            )}
+            {showFieldsSet.has(FIELD_NAMES.targetPropertyTokenOverride) && (
+              <DataElementSelector>
+                <FormikTextField
+                  data-test-id={FIELD_NAMES.targetPropertyTokenOverride}
+                  label="Target property token"
+                  name={`${prefix}.com_adobe_target.propertyToken`}
+                  description="The token for the destination property in Adobe Target. The token set here overrides the one set in your datastream configuration."
+                  width="size-5000"
+                />
+              </DataElementSelector>
+            )}
+            {showFieldsSet.has(FIELD_NAMES.reportSuitesOverride) && (
+              <ReportSuitesOverride prefix={prefix} />
+            )}
+          </Flex>
+        </FieldSubset>
+      </FormElementContainer>
+    </>
+  );
+};
+
+Overrides.propTypes = {
+  instanceFieldName: PropTypes.string,
+  largeHeader: PropTypes.bool,
+  showFields: PropTypes.arrayOf(PropTypes.oneOf(Object.values(FIELD_NAMES)))
+};
+
+export default Overrides;
```

Comment:
Somewhere we should indicate that the overrides have to be specified in the allow-list within the datastream configuration. I could see lots of people being surprised by this.